src/databricks/labs/ucx/assessment/aws.py

```diff
@@ -72,6 +71,10 @@ class AWSResources:
     S3_ACTIONS: typing.ClassVar[set[str]] = {"s3:PutObject", "s3:GetObject", "s3:DeleteObject", "s3:PutObjectAcl"}
     S3_READONLY: typing.ClassVar[str] = "s3:GetObject"
     S3_REGEX: typing.ClassVar[str] = r"arn:aws:s3:::([a-zA-Z0-9+=,.@_-]*)\/\*$"
+    UC_MASTER_ROLES_ARN: typing.ClassVar[list[str]] = [
+        "arn:aws:iam::414351767826:role/unity-catalog-prod-UCMasterRole-14S5ZJVKOTYTL",
```

Comment:
We should not have those kind of informations in public repos

Reply:
Public information. No need to hide.
If we want to hide, we can hash.

Reply:
Link to public doc to be sure 😜😉

Reply:
https://docs.databricks.com/en/connect/storage/tutorial-s3-instance-profile.html